schema/deploy/todo_app.sql

```diff
@@ -0,0 +1,8 @@
+-- Deploy todo:todo_app to pg
+
+BEGIN;
+
+-- XXX Add DDLs here.
+CREATE SCHEMA if not exists todo_app;
+
+COMMIT;
```

Comment:
The team's coding style is to have all SQL in lowercase

Reply:
Will update this.